ui/component/settingsStripeCard/view.jsx

```diff
@@ -70,6 +70,7 @@ const SettingsStripeCard = (props: Props) => {
   const [cardNameValue, setCardNameValue] = React.useState('');
   const [isLoading, setLoading] = React.useState(false);
   const [formError, setFormError] = React.useState();
+  const [stripeBillingEmail, setStripeBillingEmail] = React.useState('');
```

Comment:
:small_red_triangle: Incorrect use of `useState`

State is not required for derivative of props.  Just use the prop directly in this case.

```
const stripeBillingEmail = cardDetails ? cardDetails.email : ''

or 

navigate={`${STRIPE.STRIPE_BILLING_URL}?prefilled_email=${cardDetails?.email}`}
```